.projenrc.js

```diff
@@ -148,16 +147,17 @@ const project = new web.ReactTypeScriptProject({
       },
     },
   ];
-  project.buildWorkflow.addJobs({
-    cypress: {
-      name: "E2E Tests",
-      runsOn: "ubuntu-latest",
-      permissions: {
-        checks: "write",
-        contents: "read",
-      },
-      steps: cypressRunSteps,
+
+  const integWorkflow = project.github.addWorkflow("integ");
```

Comment:
The latest version of projen doesn't offer an `addJob` method on `buildWorkflow`, instead it offers `addPostBuildJob`, which is intended for jobs that requires the build artifact. 

These tests don't actually need anything from the build job, so decided to move it to a separate workflow. Similarly to how we do it in [projen](https://github.com/projen/projen/blob/main/.github/workflows/integ.yml) itself.